invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCitationField.js

```diff
@@ -51,25 +51,26 @@ export class RecordCitationField extends Component {
     return <Message negative>{message}</Message>;
   };
 
-  fetchCitation = async (record, style) => {
-    return await axios(
-      `${record.links.self}?locale=${navigator.language}&style=${style}`,
-      {
-        headers: {
-          Accept: "text/x-bibliography",
-        },
-      }
-    );
+  fetchCitation = async (record, style, includeDeleted) => {
+    const includeDeletedParam = includeDeleted ? "&include_deleted=1" : "";
+    const url = `${record.links.self}?locale=${navigator.language}&style=${style}${includeDeletedParam}`;
```

Comment:
did you test this change with both "0" and "1" ? Are they string values?
my feeling is that with this approach if someone passess `include_deleted` with value `"0"` (string), you will likely end up with js evaluation of the string, which will be true. see:
![Screenshot 2023-09-27 at 14 00 18](https://github.com/inveniosoftware/invenio-app-rdm/assets/38131488/5a3de231-33b9-4286-8a68-1759cae74e6b)

Reply:
The value `include_deleted` is parsed in the view by the `pass_include_deleted` decorator which returns a boolean and then passed down to the cmp and the cmp has set the Proptype of `includeDeleted` as `bool`, so I would say that we are quite safe, however someone could reuse the cmp wronlgy (as any other cmp I guess), therefore I have applied Zachs suggetion above.